polartools/load_data.py

```diff
@@ -10,13 +10,13 @@
 try:
     from databroker.queries import TimeRange
 except ModuleNotFoundError:
-    pass
+    pass 
 
 # TODO: For the beamline we can import db directly, and not worry about it
 # all over the place here.
 
 
-def load_spec(scan_id, file_name, folder=''):
+def load_spec(scan_id, file_name, folder='',**kwargs):
```

Comment:
Why is the `**kwargs` is needed?

Reply:
It did not work without for me. Have to check again.